Sources/Rendering/Core/Actor2D/test/testActor2D.js

```diff
@@ -40,7 +40,7 @@ test.onlyIfWebGL('Test Actor2D', (t) => {
 
   const actor2D = gc.registerResource(vtkActor2D.newInstance());
   actor2D.getProperty().setColor([0.5, 0.8, 0]);
-  actor2D.getProperty().setOpacity(0.3);
+  actor2D.getProperty().setOpacity(0.5);
```

Comment:
I'm semi-surprised that this didn't warrant a test image change. I suppose pixel matching tolerance is high enough?